### ISSUE TITLE
Removal of AV Update from SuperCat

### ISSUE THREAD

**Issue by Twist592** [SuperCATDevelopers/SuperCAT#71]
As we discussed on 1 Feb I would like to propose that we remove the Update Antivirus (McAffee) function from SuperCat.

The script currently accomplishes the following in relation to AV.
1. Collects Windows Defender Logs from the host.
2. Collects MCaffee logs from the host.
3. Runs McAffee from the DVD (Standalone) and generates the results .

This gives us two different AV evaluations on systems that use Windows defender and an additional scan for Mcaffee that is not host based.

We currently do not have a function for updating Windows defender in the script and since this function is already being accomplished it is not necessary to add it.

The bottom line is that updating the AV definitions on the PC are not a core function of SuperCat and if we want to remove them from the script we should do so prior to releasing version 1 of SuperCat.

Dale
